app/services/moves/importer.rb

```diff
@@ -56,7 +55,10 @@ def import_moves
           end
         else
           new_count += 1
-          new_move.save!
+          new_move.save
+          unless new_move.persisted?
+            raise new_move.errors
+          end
```

Comment:
What's the intention behind this? I'm probably just missing something obvious, but wouldn't `new_move.save!` raise consistently with L54/55 for an existing move?

Reply:
That was an unintentional commit that I thought I had removed